vic/drivers/shared_image/src/vic_store.c

```diff
@@ -1266,6 +1266,9 @@ vic_store(dmy_struct *dmy_state,
 
     free(ivar);
     free(dvar);
+
+    // free memory
```

Comment:
remove whiteline (1269) and comment (1270) - the previous lines also free memory so this is nothing different